Dao/TagsDao.php

```diff
@@ -224,6 +224,36 @@ public function deleteContainerTag($idSite, $idContainerVersion, $idTag, $delete
         Db::query($query, $bind);
     }
 
+    /**
+     * @param int $idSite
+     * @param int $idContainerVersion
+     * @param int $idTag
+     */
+    public function pauseContainerTag($idSite, $idContainerVersion, $idTag)
+    {
+        $table = $this->tablePrefixed;
+
+        $query = "UPDATE $table SET status = ? WHERE idsite = ? and idcontainerversion = ? and idtag = ? and status != ?";
+        $bind = array(self::STATUS_PAUSED, $idSite, $idContainerVersion, $idTag, self::STATUS_PAUSED);
+
+        Db::query($query, $bind);
+    }
+
+    /**
+     * @param int $idSite
+     * @param int $idContainerVersion
+     * @param int $idTag
+     */
+    public function resumeContainerTag($idSite, $idContainerVersion, $idTag)
+    {
+        $table = $this->tablePrefixed;
+
+        $query = "UPDATE $table SET status = ? WHERE idsite = ? and idcontainerversion = ? and idtag = ? and status != ?";
+        $bind = array(self::STATUS_ACTIVE, $idSite, $idContainerVersion, $idTag, self::STATUS_ACTIVE);
```

Comment:
I wonder if this should look for status paused so that there isn't a chance of activating a deleted tag?

```suggestion
        $query = "UPDATE $table SET status = ? WHERE idsite = ? and idcontainerversion = ? and idtag = ? and status = ?";
        $bind = array(self::STATUS_ACTIVE, $idSite, $idContainerVersion, $idTag, self::STATUS_PAUSED);
```

Reply:
@snake14 This makes more sense :+1:

Reply:
Added a test  `test_shouldNotResumeDeletedContainerTag` to test this behaviour.